sky/provision/__init__.py

```diff
@@ -186,12 +186,12 @@ def get_cluster_info(
 def get_command_runners(
     provider_name: str,
     cluster_info: common.ClusterInfo,
-    **crednetials: Dict[str, Any],
+    **credentials: Dict[str, Any],
```

Comment:
Thanks for fixing this in #3924 :) perhaps this branch needs to rebased/merged with latest?